README.md

```diff
@@ -1,2 +1,59 @@
-# JS-TO-DO
-This repository contains source of a To-Do List developed in JavaScript.
+# TO-DO LIST
+
+> This repository contains source of a To-Do List.
+ User can add projects.
+ User can add tasks to projects.
+ User can edit a task or delete it.
+
+![screenshot](./src/screenshot.png)
+
+## Set Up Project Locally
+
+**To get this project set up on your local machine, follow these simple steps:**
+
+**Step 1**<br>
+Navigate through the local folder where you want to clone the repository and run<br>
+`git@github.com:SajjadAhmad14/JS-TO-DO.git` to clone the repo to your local folder.<br>
+or with https<br>
+`https://github.com/SajjadAhmad14/JS-TO-DO.git`.<br>
+**Step 2**<br>
+Run `cd JS-TO-DO`<br>
+**Step 2**<br>
+Run `npm install`<br>
+
```

Comment:
## Add a live link 
- [ ] Please add a live link to the page on this project's `README` file. It will make the project's documentation complete as this is a mandatory must-have in line with having an all-round descriptive `README.md` just as explained in this  [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)